src/main/scala/rna16s.scala

```diff
@@ -46,7 +46,10 @@ case object rna16s extends AnyBlastDB {
   /* Here we want to keep sequences which */
   val predicate: (Row, FASTA.Value) => Boolean = { (row, fasta) =>
      /* - are annotated as encoding 16S */
-     (row.select(gene_name) == geneNameFor16S) &&
+     (row.select(gene_name) == geneNameFor16S ||
+      fasta.getV(header).description.toLowerCase.contains("16s") ||
+      fasta.getV(header).description.toLowerCase.contains("small subunit ribosomal rna")
```

Comment:
Then my question is:
The 16S sequences from SILVA and Greengenes that only include in the header of the fasta sequence **rrn** or **rRNA**, are all of them annotated as rRNA and are selected for our database?